Updating Metrics Summary with Facet Panel (#10568)
DataDog/documentation: content/en/metrics/summary.md

## Overview

The [Metrics Summary page][1] displays a list of your metrics reported to Datadog under a specified time frame: the past hour, day, or week. 

Search your metrics by name or tag using the **Metric** or **Tag** search fields:

{{< img src="metrics/summary/tagexplorer2.gif" alt="Filter by Tag"  style="width:75%;">}}

## Facet Panel

> The search bars provide the most comprehensive set of actions to filter the list of metrics. But we also offer facets to quickly filter your metrics by:

Reviewer: ```suggestion
The search bars provide the most comprehensive set of actions to filter the list of metrics. But facets can also quickly filter your metrics by:
```

* **Configuration** : Quickly identify metrics with tag configurations or additional percentile aggregations

> * **Metric Type** : Quickly identify distributions vs non-distributions (counts,gauges,rates)

Reviewer: ```suggestion
* **Metric Type** : Quickly identify distributions vs non-distributions (counts, gauges, rates)
```
